packages/api-v4/src/images/types.ts

```diff
@@ -16,12 +30,14 @@ export interface Image {
   type: string;
   is_public: boolean;
   size: number;
+  total_size: number;
```

Comment:
Optional: It might be nice to add some JSDoc style comments to the fields

```suggestion
  /**
   * The total storage consumed by the Image across its regions.
   */
  total_size: number;
```